README.md

```diff
@@ -2,103 +2,85 @@
 	<img height="128px" src="https://github.com/kixelated/moq-rs/blob/main/.github/logo.svg" alt="Media over QUIC">
 </p>
 
-Media over QUIC (MoQ) is a live media delivery protocol utilizing QUIC streams.
+Media over QUIC (MoQ) is a live media delivery protocol utilizing QUIC.
+It's a client-server model (not peer-to-peer) that is designed to scale to enormous viewership via clustered relay servers (aka a CDN).
+The application determines the trade-off between latency and quality, potentially on a per viewer-basis.
+
 See [quic.video](https://quic.video) for more information.
+Note: this project uses a forked implementation of the IETF standardization effort.
```

Comment:
It's probably worth pointing to the drafts here and noting where folks can find implementations of the MoQ WG drafts. For example, I intend to maintain a fork at  [englishm/moq-rs](https://github.com/englishm/moq-rs) that tracks [draft-ietf-moq-transport](https://datatracker.ietf.org/doc/draft-ietf-moq-transport/).